cmd/sidecar/main.go

```diff
@@ -30,33 +30,34 @@ func main() {
 	tokenFilePath := flag.String("t", "/run/conjur/access-token",
 		"Path to Conjur access token")
 
-	configAuthn, err := authnConfig.NewFromEnv(clientCertPath, tokenFilePath)
+	// Create new Storage
+	configStorage, err := storageConfig.NewFromEnv(tokenFilePath)
 	if err != nil {
 		errLogger.Printf(err.Error())
 		os.Exit(1)
 	}
 
-	// Create new Authenticator
-	authn, err := authenticator.New(*configAuthn)
+	storageHandler, err := storage.NewStorageHandler(*configStorage)
```

Comment:
 I didn't want to change the secretConfig since i knew that you are working on it in your PR and maybe it should be merging with the storageConfig somehow

Reply:
cool so i'll take care of it